package.json

```diff
@@ -10,6 +10,7 @@
   "keywords": [],
```

Comment:
There seems to be a lot more stuff here that have been added, than what is actually needed.